tools/clang/test/HLSLFileCheck/hlsl/control_flow/attributes/unroll/unroll_switch_exit_crash.ll

```diff
@@ -0,0 +1,121 @@
+; RUN: %opt %s -dxil-loop-unroll -S | FileCheck %s
```

Comment:
It looks like we could probably simplify this bitcode quite a bit? Would be nice to have a minimal loop with the switch here to demonstrate the problem. There is lots of dxil-isms and debug info that seem unnecessary to trigger the bug.

Reply:
Simplified it!